extensions/wikia/DesignSystem/styles/design-system-notifications.scss

```diff
@@ -0,0 +1,206 @@
+  overflow: hidden;
+  position: absolute;
+  width: 0;
+}
+
+.wds-notifications {
+  &__mark-all-as-read-button {
+	@include justify-content(flex-start);
+	box-shadow: 0 -1px 10px 0 rgba(0, 0, 0, 0.1);
+	display: none;
+	min-height: 44px;
+	padding: 0;
+	width: 100%;
+
+	&.wds-is-visible {
+	  @include flexbox();
+	}
+  }
+  &__mark-all-as-read {
+	color: $wds-color-link;
+	font-size: $wds-typescale-size-minus-2;
+	font-weight: bold;
+	padding: 18px;
+	text-transform: uppercase;
+	cursor: pointer;
+  }
+
+  &__dropdown-content {
+	@include flex-direction(column);
+	@include justify-content(center);
+	@include align-items(center);
+	color: $notifications-dropdown-content-color;
+	font-size: $wds-typescale-size-minus-1;
+	min-width: 18px * 2 + 320px;
+	width: 100%;
+	white-space: normal;
+  }
+
+  &__notification-list {
+	box-sizing: border-box;
+	max-height: 420px;
+	overflow-y: auto;
+	width: 100%;
+  }
+
+  &__zero-state {
+	padding: 18px 18px 4px 18px;
+	display: none;
+	&.wds-is-visible {
+	  display: block;
+	}
+  }
+}
+
+$notification-card-text-color: $wds-color-dark-blue-gray;
+$notification-card-anchor-color: $notifications-dropdown-content-color;
+
+.wds-notification-card {
+  padding: 12px 0 8px;
+  text-align: left;
+  .wds-notification-card__outer-body {
+	cursor: pointer;
+	display: flex;
+  }
+
+  &__icon-wrapper {
+	@include flexbox();
+	cursor: pointer;
+	padding-left: 12px;
+	width: 30px;
+
+	.wds-icon {
+	  fill: $wds-icon-color;
+	  width: 18px;
+	  height: 18px;
+	  min-width: inherit;
+	}
+	.wds-is-unread {
+	  fill: $wds-color-link;
+	}
+  }
+
+  &__body {
+	@include flexbox();
+	@include flex-direction(column);
+	@include flex-grow(1);
+	@include justify-content(flex-start);
+	padding-left: 12px;
+	padding-right: 12px;
+  }
+
+  &__text {
+	color: $notification-card-text-color;
+	font-size: $wds-typescale-size-minus-1;
+	line-height: 18px;
+	margin: 0;
+	b {
+	  font-weight: bold
+	}
+  }
+
+  &__snippet {
+	color: $wds-color-blue-gray;
+	font-size: $wds-typescale-size-minus-1;
+	font-style: italic;
+	line-height: 18px;
+	margin-top: 8px;
+  }
+
+  &__context-list {
+	@include flexbox();
+	color: $wds-color-blue-gray;
+	font-size: $wds-typescale-size-minus-2;
+	font-weight: bold;
+	list-style-type: none;
+	margin: 0;
+	padding: 8px 0 0;
+  }
+
+  &__context-separator {
+	margin: 0 8px 0 8px;
+  }
+
+}
+
+.wds-avatar-stack {
+  @include flexbox();
+  @include align-items(center);
+  margin: 8px 18px 6px 40px;
+
+  &__avatar {
+	color: $wds-color-light-c4;
+  }
+
+  @for $i from 1 to 6 {
+	.wds-avatar-stack__avatar:nth-child(#{$i}) {
+	  position: relative;
+	  transform: translate(($i - 1) * - 15px);
+	  z-index: 5 - $i;
+	}
+  }
+
+  &__overflow {
+	padding-left: 6px;
+	background-color: $wds-color-white;
+	border: 0;
+	color: $wds-color-light-c6;
+	font-size: $wds-typescale-size-minus-2;
+	transform: translate(-60px);
+	width: auto;
+  }
+}
+
+$duration: 1s;
+$spinner-color: $wds-color-link;
+
+.wds-spinner {
```

Comment:
Why spinner is defined in notifications file? Should we define it in Design System repository and include it here?

While we are introducing new component that should start replacing old spinners, please consider knowledge sharing so other developers are aware which solution is preferred. Additionally we could mark old spinners as deprecated.